tests/tests/kakarot_api.rs

```diff
@@ -126,3 +133,40 @@ async fn test_kakarot_get_starknet_transaction_hash_with_none_tx_hash(#[future]
 
     drop(server_handle);
 }
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_kakarot_get_config(#[future] katana: Katana, _setup: ()) {
+    std::env::set_var("STARKNET_NETWORK", "http://0.0.0.0:1010");
+    std::env::set_var(
+        "WHITE_LISTED_EIP_155_TRANSACTION_HASHES",
+        "0xe65425dacfc1423823cb4766aa0192ffde61eaa9bf81af9fe15149a89ef36c28",
+    );
+    let starknet_config = KakarotRpcConfig::from_env().expect("Failed to load Kakarot RPC config");
+    let expected_constant = Constant {
+        max_logs: *MAX_LOGS,
+        starknet_network: String::from(starknet_config.network_url),
+        retry_tx_interval: get_retry_tx_interval(),
+        transaction_max_retries: get_transaction_max_retries(),
+        max_felts_in_calldata: *MAX_FELTS_IN_CALLDATA,
+        white_listed_eip_155_transaction_hashes: get_white_listed_eip_155_transaction_hashes(),
+    };
```

Comment:
The expected value doesn't need to call the functions, it can just be hardcoded values. The way I would write the test is:
- init all the environment variables.
- set the `Constant` instance with these values.
- rest of the test